dask_planner/src/sql/types.rs

```diff
@@ -120,6 +120,10 @@ impl DaskTypeMap {
         })
     }
 
+    fn __str__(&self) -> String {
+        format!("{:?}", self.sql_type)
```

Comment:
This doesn't need to be a "Debug" print statement but no harm in leaving it this way.

Reply:
Yeah mostly did this to get around the need to implement `Display` for `SqlTypeName`, without which we end up getting a compiler error